src/com/spikes2212/genericsubsystems/drivetrains/commands/DriveTank.java

```diff
@@ -7,70 +7,79 @@
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
- * This command moves a {@link TankDrivetrain} by giving each side a different speed.
- *
- * @author Unknown
+ * This command moves a {@link TankDrivetrain} using left and right speeds.
  */
 public class DriveTank extends Command {
 
 	protected final TankDrivetrain tankDrivetrain;
 	protected final Supplier<Double> leftSpeedSuplier;
 	protected final Supplier<Double> rightSpeedSuplier;
 
-    /**
-     * This constructs a new {@link DriveTank} command
-     * that moves each of the given {@link TankDrivetrain}'s sides using a given speed.
-     *
-     * @param drivetrain the drivetrain this command requires and moves.
-     * @param leftSpeed  the speed to move the left side with.
-     * @param rightSpeed the speed to move the right side with.
-     */
-    public DriveTank(TankDrivetrain drivetrain, double leftSpeed, double rightSpeed) {
-        // Use requires() here to declare subsystem dependencies
-        // eg. requires(chassis);
-        this(drivetrain, () -> leftSpeed, () -> rightSpeed);
-    }
+	/**
+	 * This constructs a new {@link DriveTank} command that moves the given
+	 * {@link TankDrivetrain} acording to constant left and right speeds.<br>
+	 * Positive values move forwards.
```

Comment:
maybe you should call the speeds "leftside" and "rightside" speeds.
also - fix acording to according